### DISCUSSION TITLE
Struggling with sync_list and business shared directories

### DISCUSSION THREAD

**Discussion by nicokist** [abraunegg/onedrive#3002] [Q&A]
Dear all:

I have thoroughly searched the documentation but I am still confused by this. My reading is that if I have the following sync_list file:
```
*.xlsx
*.xlsm
```

I should expect it to download "Business_shared_folder/test.xlsx" and Business_shared_folder/subfolder/test2.xlsx; however, instead I get the following message:

```
Skipping shared folder shortcut - excluded by sync_list config
```

This can be resolved by removing sync_list or putting 'Business_shared_folder' into the sync list, however then all files in Business_shared_folder, including those with different extensions, are downloaded. 

I have also tried the following sync_list file:

```
./Business_shared_folder/*.xlsx
```

Which I would expect to work, however that also leads to the message 'Skipping shared folder shortcut - excluded by sync_list config'.

I am on the latest v2.5.3-8-g5d45e24, running inside Docker.

Link to some earlier discussion: [#3000](https://github.com/abraunegg/onedrive/issues/3000#issuecomment-2495669309)

**Comment by abraunegg** [comment]
Please raise a new bug report

**Reply by nicokist** [reply]
Done! #3003 

Thanks @abraunegg !
